### DISCUSSION TITLE
Property attributes in records

### DISCUSSION THREAD

**Discussion by tuscen** [dotnet/csharplang#6044] [Q&A] [answered]
Is it possible to place an attribute on a generated property without writing a record body?

E.g. with this code `PropertyAttribute` is placed only on the constructor's parameter and not on the generated property.

Thanks.

```csharp
[AttributeUsage(AttributeTargets.Property | AttributeTargets.Parameter)]
public class PropertyAttribute : Attribute
{ }

/// <summary>
///
/// </summary>
/// <param name="Bar"></param>
public record Entity([Property] string Value);

```

**Comment by RikkiGibson** [comment, marked as answer]
You should be able to say `record Entity([property: SomeAttribute] string Value);`.

**Reply by HaloFour** [reply]
Yep, see: [SharpLab](https://sharplab.io/#v2:EYLgtghglgdgPgAQAwAIEEYB0BhA9mAB1xgFMYAXAWVwBMSAbAbgFgAoNgJxIGNcOaUABRIcAzsQAUAbQIdcBEeQCeIFABESo7hygFyUSQCIAYrlyGAlAF006VADkIYEhcZA)

**Reply by huoyaoyuan** [reply]
https://github.com/dotnet/roslyn/issues/58422
